storage/replica.go

```diff
@@ -1660,3 +1677,49 @@ func (r *Replica) maybeAddToSplitQueue() {
 		r.store.splitQueue.MaybeAdd(r, r.store.Clock().Now())
 	}
 }
+
+// processReplicaCmd get a raft command from command queue and
+// call processRaftCommand to execute in a loop until there is
+// not any command in the queue for 5 seconds.
```

Comment:
on top of generally saving goroutines, we've put some effort in a while back to not have any goroutines attached to `*Replica` since replicas don't have a shutdown mechanism. With the timeout, they'll "gc" themselves after a short time (but we might want to re-add `Quiesce` functionality at some point, but I'm ok with not doing it for this change).